packages/faustwp-cli/utils/sendTelemetryData.ts

```diff
@@ -0,0 +1,47 @@
+import fetch from 'node-fetch';
+import { TelemetryData } from './marshallTelemetryData';
+
+const GA_TRACKING_ENDPOINT = 'http://www.google-analytics.com/debug/collect';
+const GA_TRACKING_ID = 'G-KPVSTHK1G4';
```

Comment:
You also need to include the secret query parameter as well:

https://developers.google.com/analytics/devguides/collection/protocol/ga4/reference?client_type=firebase

Here is an example:
https://github.com/nkowaokwu/igbo_api/blob/81581a20c2a1556b28a66d6114370f2809620fb5/src/middleware/analytics.js#L33-L36